internal/config/yaml_parser.go

```diff
@@ -0,0 +1,30 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/yaml.v3"
+)
+
+func ReadConfigFile(fileName string) (mountConfig *MountConfig, err error) {
```

Comment:
nit: we can rename it to ParseConfigFile ?

Reply:
done